eth/backend.go

```diff
@@ -89,12 +89,22 @@ func (b *backend) SendTransaction(ctx context.Context, tx *types.Transaction) er
 		method = "unknown"
 	}
 
-	if err != nil {
-		glog.Infof("\n%vEth Transaction%v\n\nInvoking transaction: \"%v\".  \nTransaction Failed: %v\n\n%v\n", strings.Repeat("*", 30), strings.Repeat("*", 30), method, err, strings.Repeat("*", 75))
+	txParams := make(map[string]interface{})
+	if err := decodeTxParams(b.abiMap[string(data[:4])], txParams, data); err != nil {
 		return err
 	}
 
-	glog.Infof("\n%vEth Transaction%v\n\nInvoking transaction: \"%v\".  Hash: \"%v\". \n\n%v\n", strings.Repeat("*", 30), strings.Repeat("*", 30), method, tx.Hash().String(), strings.Repeat("*", 75))
+	var txParamsString string
+	for name, val := range txParams {
+		txParamsString += fmt.Sprintf("%v: %v   ", name, val)
+	}
+
+	if sendErr != nil {
+		glog.Infof("\n%vEth Transaction%v\n\nInvoking transaction: \"%v\".  Transaction Inputs: \"%v\"   \nTransaction Failed: %v\n\n%v\n", strings.Repeat("*", 30), strings.Repeat("*", 30), method, txParamsString, err, strings.Repeat("*", 75))
```

Comment:
Nit: Since this statement is already pretty long perhaps we can shorten "Transaction Inputs" to "Inputs" since the rest of the statement should convey that the inputs are for a transaction.

Reply:
fixed in e89ba37